### PR TITLE
Create info.ini for jquery.inputmask

### PR DESCRIPTION
**Pull request by evilangelmd**
jquery.inputmask add

### CODE REVIEW

File: files/jquery.inputmask/update.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "packageManager": "github",
+  "name": "jquery.inputmask",
+  "repo": "RobinHerbots/jquery.inputmask",
+  "files": {
+    "basePath": "dist/min/inputmask/",
+    "include": ["jquery.inputmask.min.js"]
```
Comment (megawac) [inline files/jquery.inputmask/update.json]: does it not make more sense to include everything in that folder?

Comment (evilangelmd) [inline files/jquery.inputmask/update.json]: maybe, i don't know all syntax, i took example and edited. if somewhere i made a mistake, sorry :)

Comment (jimaek) [inline files/jquery.inputmask/update.json]: Yes, in this case you can delete the include parameter completely. We want everything in that folder

Comment (evilangelmd) [inline files/jquery.inputmask/update.json]: deleted include parameter :+1:

**Comment by jimaek** [comment]
@jsdelivrbot squash and merge

**Pull request closed without merging**